src/patternfly/components/Divider/docs/code.md

```diff
@@ -0,0 +1,13 @@
+## Overview
+
+The divider renders as an `<hr>` by default. It is possible to make the divider render as an `li` or a `div` to match the HTML5 specification and context of the divider.
+
+| Attribute | Applied to | Outcome |
+| -- | -- | -- |
+| `role="separator"` | `.pf-c-divider.li`, `.pf-c-divider.div` | Indicates that the separator is a separator. |
```

Comment:
Moved the elements before the class. We do that everywhere else. Buttons and tabs (the nav examples) are a couple of examples. 

```suggestion
| `role="separator"` | `li.pf-c-divider`, `div.pf-c-divider` | Indicates that the divider is a separator. |
```